mesheryctl/internal/cli/root/adapter/adapter.go

```diff
@@ -87,6 +87,7 @@ var (
 			// ensure the mesh's adapter is available and update adapterURL if so
 			if err = validateAdapter(mctlCfg, meshName); err != nil {
 				utils.Log.Error(err)
+				return err
```

Comment:
```suggestion
				return nil
```

Reply:
Could you validate with proposed changes ? This is the way it is done for all commands in mesheryctl commands

Reply:
@Kevin222004 any update ?